### DISCUSSION TITLE
Home Assistant: automazione con attivazione long press shelly2.5

### DISCUSSION THREAD

**Discussion by disn3** [indomus/forum#2437] [HUB: Home Assistant]
Salve a tutti,
vorrei il vostro aiuto per comandare l'accensione di un faretto RGB wifi tuya attraverso la pressione prolungata di pulsanti collegati a shelly 2.5. Ho creato un'automazione e in effetti il faretto wifi (che ha alimentazione diretta) si accende e spegne premendo a lungo uno dei pulsanti collegati allo shelly 2.5. Fin qui tutto ok, ma vorrei che, premendo a lungo i pulsanti, non si accendano una serie di faretti collegati in uscita allo shelly (singola pressione commuta la serie faretti - pressione prolungata commuta solo faretto RGB).
Ho creato una nuova automazione che, a pressione prolungata dei pulsanti, dovrebbe lasciare la serie di faretti spenti se sono spenti e accesi se sono accesi ma purtroppo non funziona, infatti i faretti si accendono/spengono ad ogni pressione prolungata.
Cosa mi consigliate?

**Comment by asmaran85** [comment]
Prova cambiando questa impostazione nell'app
![Screenshot_20230422-082041](https://user-images.githubusercontent.com/95779207/233766570-d7458f93-9b14-49d5-a458-3b6e242b6259.png)
